### PR TITLE
Added Lorem ipsume

### LINKED ISSUE #1: Edit readme
PLease, add text in readme 
"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."

### PR DESCRIPTION
**Pull request by LebedevKrll**
Issue #1 fixed. Classical Lorem Ipsum added into readme

Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-#TEST
+#TE
```
Comment (MichaelKab) [inline README.md:1]: Please dell this string

Comment (MichaelKab) [inline README.md:1]: PLease delete this string

**Review by MichaelKab — COMMENTED** [review]
Delete some string

File: README.md (hunk 1 above)
Comment (LebedevKrll) [reply README.md]: I pushed new version, check it ut please

**Pull request merged**